### PR TITLE
Converting community-wide-resources to rst

### LINKED ISSUE #221: Convert the community-wide resources section to RST
We are in the process of moving this repository to Read the Docs, and to do that we need to convert our pages from Markdown to RST. This issue is to transfer the Community-wide resources section: [https://contribute.mautic.org/community-wide-resources](https://contribute.mautic.org/community-wide-resources). You can also update and improve the content on these pages if you like - do check with the team leads first to ensure it's relevant.

You can do this in the current repo, as we've got a folder set up, `/docs`, which will be used for Read The Docs.

You can build this documentation locally by cloning this repository, and following these steps: https://github.com/mautic/user-documentation/tree/5.x?tab=readme-ov-file#build-documentation-locally. Change into the /docs folder and then (assuming you have DDEV and Docker/Colima installed) type `ddev config` and select `build/html `as the root folder when configuring DDEV. Then when complete, type `ddev start` and it'll give you the link to your local testing environment.

If you need to make a link file to add an external URL, please use the command line tool 'make link'. This will generate a reference macro and a link file for you.

For internal linking check the docs on using the [:ref: role](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-ref-role) for linking within the same document, and using [:doc: to link across pages](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role).

**Please note: if you see folders called .revs these are old version control files from Grav. Ignore them! (they should be deleted) and instead work on the actual files.**

When you've converted the file/s, **they need to live within the /docs folder**. We can create a structure using sub-folders to organise the sections. You also need to add the pages to the Table of Contents in the index.rst file for them to show on the menu. Docs on the ToC can be found [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/indices.html).
We highly recommend that you use VS Code with Vale installed. Let us know if you have any problems.

Some useful resources: 
- https://sublime-and-sphinx-guide.readthedocs.io/en/latest guide to Sphinx/RST formatting
- https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-the-external-links-extension how to do external links - always check the /docs/links directory first! You can use the terminal command '`make link`' to create the link file automagically.

There are many MD to RST tools out there which can help speed up the process, however you will still need to review your output to ensure it passes our Vale grammar rules. Headings must always be unique, as well.

We use the following for headings:

```
H1 ###
H2 ***
H3 ===
H4 ---
H5 ~~~
```

Please ask in #t-education on Slack if you get stuck with anything!

### PR DESCRIPTION
**Pull request by Iqmaa**
fixed #221 
I have been able to convert 'community wide resources' to rst format, excluding the page's metadata

### CODE REVIEW

File: docs/pages/07.community-wide-resources/.revs/20200310-170715/chapter.rst
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/.revs/20200310-170715/chapter.rst:1]: This is a revision file (indicated by .revs) so it should be ignored.

File: docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+emojis, e.g. ``:one:.:four:`` which will show up as:
+
+.. figure:: slack-emoji-numbers.png
+   :alt: Slack Emojis
+
+
+
+Your agenda might now look something like this:
+
+::
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:105]: This markup looks a little wonky.

**Review by RCheesley — CHANGES_REQUESTED** [review]
Hey there! Thanks for making a great start on this!

A few things that you need to factor into this PR before we can merge - these are all documented in the issue - please take the time to read it:

1. Please follow our headings syntax, as outlined in the issue description. We don't use over and under headings, only under.
2. Please don't touch the .revs folder - if anything they should be deleted. It's the old version control system Grav used, and not the live files.
3. You've generated link files but you're not using them properly, please check the docs here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-the-external-links-extension
4. We don't need to use random lines like you've done in the async meetings file on line 5 - they don't serve a purpose, so you can remove them.
5. Please check that all links are using the link macro, e.g. on line 59 of the same file there's a link posted.
6. Please use ref: or doc: to link to other pages.
7. Please use the image markup here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/images.html#add-images rather than figure.

Let me know if you are stuck with anything!

**Comment by Iqmaa** [comment]
Hey @RCheesley, sorry for that.
I have:

1. Fixed the headings issue
2. Converted the .rev files back to original form
3. Corrected all linking implementation issues
4. removed unnecessary lines
5. and replaced the figure with the image markup

Please let me know if there are any other changes I need to make.

**Comment by Iqmaa** [comment]
Hey @RCheesley, sorry for that.
I have:

1. Fixed the headings issue
2. Converted the .rev files back to original form
3. removed unnecessary lines
4. and replaced the figure with the image markup

But I am having an issue with the external linking. Although I have implemented the changes according to the instructions the links aren't showing as they should be, do you perhaps know why this is?

File: docs/pages/07.community-wide-resources/.revs/20200310-170715/chapter.md
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/.revs/20200310-170715/chapter.md:1]: @Iqmaa these don't need to be pages/* any more, in fact in the index.rst we've declated that they're in resources/* so I will move these for you and make a commit directly, to save us having to go through those changes. Hold on to your hat :)

File: docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst [hunk 2]
```diff
@@ -0,0 +1,273 @@
+How to prepare & run a Mautic Team Meeting
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:1]:
```suggestion
How to prepare & run a Mautic team meeting
```

File: docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+==========================================
+
+Meeting cycle; Async vs. Live meetings
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:4]:
```suggestion
Meeting cycle; async vs. live meetings
```

File: docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+possible, so that we have the best chances of offering a convenient slot
+for people in most parts of the world at least every 2nd time.
+
+Always remember to avoid barriers for potential new volunteers (which
+you are probably not aware of) !
+
+Preparing for a meeting
+=======================
+
+High-Level Agenda
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:42]:
```suggestion
High-level agenda
```

File: docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+- “swap 1 and 2” 
+- “1 is not needed because we have it all in Trello” 
+- “3 doesn’t make send for us, but xyz does” 
+- “in 2 we only look at select items in Trello, not the entire backlog” 
+- …
+
+Don’t forget: Please do experiment! Let the structure improve over time,
+and of course: Adopt to temporary circumstances.
+
+Specific Topics within the Agenda
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:66]:
```suggestion
Specific topics within the agenda
```

File: docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst [hunk 6]
```diff
@@ -0,0 +1,273 @@
...
+and
+
+1. Review last meeting’s (no-Trello) notes & actions 1.1 canva.com 1.2
+   Places of Discussion (Slack vs. Trello vs. Forum vs. Github) 1.3
+   Motion to remove Trello integration from t-community
+
+2. Review Trello board 2.1 Ensure all teams have an onboarding
+   person/workflow …
+
+Make Agenda items stick out in Slack
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:88]:
```suggestion
Make agenda items stick out in Slack
```

File: docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst [hunk 7]
```diff
@@ -0,0 +1,273 @@
...
+
+   :one: Review last meeting’s (no-Trello) notes & actions 
+   :one:.:one: canva.com
+   :one:.:two: Places of Discussion (Slack vs. Trello vs. Forum vs. Github)
+   :one:.:three: Motion to remove Trello integration from t-community
+
+   :two:  Review Trello board
+   :two:.:one: Ensure all teams have an onboarding person/workflow
+   ...
+Post meeting invitation and Agenda in Slack
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:111]:
```suggestion

Post meeting invitation and Agenda in Slack
```

Usually code blocks require a clean line break after them.

File: docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst [hunk 8]
```diff
@@ -0,0 +1,273 @@
...
+
+   Hey everyone, we have our #t-community team meeting tomorrow, Thursday:
+     UK time: 11:00 am
+     Your time: https://everytimezone.com/s/12345 
+   This will be a live meeting, everyone who can make it is very welcome to attend in person via Audio (optionally Video): https://meet.jit.si/mautic-community-team
+
+   For continuation (and also for everybody who can not make it to the live call) we will turn every meeting topic into a thread right here in Slack, so you can still join the asynchronous follow-up discussion whenever you are available and as often as you can, within the next 24 hours.
+   /cc @teammember1 @teammember2 @teammember3 ...
+
+Prepare your Meeting content
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:152]:
```suggestion
Prepare your meeting content
```

File: docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst [hunk 9]
```diff
@@ -0,0 +1,273 @@
...
+everyone… Welcome to our team meeting!” post, with the “Also send to
+#name of channel” checkbox ticked, something like Thanks everyone! This
+meeting is now officially over and I am moving the content to the forum.
+
+Moving the content to the forum
+===============================
+
+Use the :doc:`meeting parser tool </community-wide-resources/meetings/asynchronous-async-meetings#exporting-slack-meetings>`
+
+Live Meetings
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:266]:
```suggestion
Live meetings
```

File: docs/pages/07.community-wide-resources/02.meetings/chapter.rst [hunk 10]
```diff
@@ -0,0 +1,8 @@
+# Meetings
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/02.meetings/chapter.rst:1]:
```suggestion
Meetings
```

Might ned to correct the underline heading once tweaking this.

File: docs/pages/07.community-wide-resources/03.avoiding-burnout/docs.rst [hunk 11]
```diff
@@ -0,0 +1,279 @@
...
+frightening stuff, and unfortunately it appears to be prevalent.
+
+Some of you will have read the list and identified with many of these
+steps, whereas some of you will be identifying others who may have
+exhibited some of the steps. I have met and known people who have
+exhibited almost all the behaviors described in these steps, and when
+serious burnout takes its grip, it can destroy families, careers, and
+many other aspects of life.
+
+Detecting and Treating Burnout
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/03.avoiding-burnout/docs.rst:134]:
```suggestion
Detecting and treating burnout
```

File: docs/pages/07.community-wide-resources/03.avoiding-burnout/docs.rst [hunk 12]
```diff
@@ -0,0 +1,279 @@
...
+Although most communities are firmly wedged in the volunteer category,
+that doesn’t mean that their participants don’t develop, feel, and react
+to stress. The lack of compulsion behind volunteers’ involvement and
+contribution does not mean that volunteers who feel stress can just go
+and do something else. People grow attached to communities, their ethos,
+and their sense of family. The involvement may not be contractually
+required, but it is often emotionally required inside the mind of the
+contributor.
+
+Work/Life Balance
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/03.avoiding-burnout/docs.rst:203]:
```suggestion
Work/life balance
```

File: docs/pages/07.community-wide-resources/03.avoiding-burnout/docs.rst [hunk 13]
```diff
@@ -0,0 +1,279 @@
...
+friend. It is incredible how those few days with a friendly face,
+getting out in the countryside, having a few drinks, and getting away
+from a computer helped.
+
+If you suspect you or someone else is burning out, tell him to do the
+same and get away for a few days. He will almost certainly claim he
+can’t or doesn’t need to, but stand firm: it is for his own good, and he
+will thank you for it.
+
+VOLUNTEERISM ESCAPES NOTHING:
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/03.avoiding-burnout/docs.rst:190]:
```suggestion
Volunteerism escapes nothing:
```

File: docs/pages/07.community-wide-resources/04.getting-help-in-a-crisis/docs.rst [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+combined with drugs / alcohol, and since we are all more closely
+connected to one another we may be aware of a problem sooner than those
+physically close to someone, or their family.
+
+If you are worried about the mental health of somebody in the community,
+reach out to them immediately to let them know that you care about them
+and that you are concerned. Then encourage them to contact one of the
+resources listed here to get the professional help that they need.
+
+Prevention Hotlines
```
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/04.getting-help-in-a-crisis/docs.rst:17]:
```suggestion
Prevention hotlines
```

File: docs/pages/07.community-wide-resources/chapter.rst
Comment (RCheesley) [inline docs/pages/07.community-wide-resources/chapter.rst:1]: This one isn't required, we don't need a chapter any more.

**Review by RCheesley — CHANGES_REQUESTED** [review]
A few changes but mostly the same things:

- Not using camel case in headings
- A file not required
- A missed link macro

I will make a commit to move all the files to the right folder so you don't need to worry about that, and will also write some half-decent onboarding docs on how to write for the docs as I appreciate this is somewhat lacking!

**Comment by RCheesley** [comment]
LMK when you're done @Iqmaa if you'd like me to fix the RST issues/move the directories!

**Comment by RCheesley** [comment]
OK Vale is now happy (I have checked it locally, as the linter is having a problem which I'll try to fix in another PR) - the build is now passing. LGTM! Thanks so much for getting this underway @Iqmaa and for your patience! 🚀

**Comment by Iqmaa** [comment]
I'm really grateful for your patience and help @RCheesley, I learnt quite a lot from contributing to this.
Thanks again !

**Pull request merged**